src/config.rs

```diff
@@ -246,6 +246,7 @@ impl Frame {
 pub struct HookBuilder {
     filters: Vec<Box<FilterCallback>>,
     capture_span_trace_by_default: bool,
+    panic_note: Option<String>,
```

Comment:
Instead of requiring a `String` lets allow any `Display` types

```suggestion
    panic_note: Option<Box<dyn Display + Send + Sync + 'static>>,
```